receiver/prometheusreceiver/factory.go

```diff
@@ -56,6 +63,7 @@ func createMetricsReceiver(
 	nextConsumer consumer.Metrics,
 ) (receiver.Metrics, error) {
 	configWarnings(set.Logger, cfg.(*Config))
+	addDefaultFallbackScrapeProtocol(cfg.(*Config))
```

Comment:
Note for after this merges (non-blocking): If this is setting the default scrape protocol, why do we still need to set it in our configuration everywhere?

Reply:
By everywhere you mean all those test changes I've made?

Reply:
That, and also in the compliance repo